graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/util/Uuids.java

```diff
@@ -0,0 +1,225 @@
+package io.stargate.graphql.schema.schemafirst.util;
```

Comment:
This class misses the Stargate authors notice.. Also a link where did you copy this from would be nice (hope you were not writing this from scratch)